css/main.css

```diff
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  width: 1200px;
+  margin: 0 auto;
+}
+
+.header_page {
+  height: 100px;
+  width: 100%;
+  min-width: 1200px;
+  background: red;
+}
+
+.left_block {
+  float: left;
+  width: 10%;
+  height: 100%;
+}
+
+.right_block {
+  float: right;
+  width: 10%;
+  height: 100%;
+}
+
+.header_content {
+  margin-left: 10%;
+  margin-right: 10%;
+  height: 100%;
+  display: table;
```

Comment:
it's one of the way, but you have only one line text
please think about another solution, thaks